crates/ra_ide_api/src/lib.rs

```diff
@@ -243,9 +241,7 @@ impl<T> RangeInfo<T> {
 
 #[derive(Debug)]
 pub struct CallInfo {
```

Comment:
I've never been terribly happy with `CallInfo`. Maybe it should be `CallSite`? Just a thought and doesn't need to change in this PR.